src/test/java/teammates/test/cases/webapi/SaveFeedbackQuestionActionTest.java

```diff
@@ -269,6 +269,34 @@ public void testExecute_invalidQuestionNumber_shouldThrowException() {
                 logic.getFeedbackQuestion(typicalQuestion.getId()).getQuestionDescription());
     }
 
+    @Test
+    public void testExecute_invalidRecommendedLength_shouldThrowException() {
+        InstructorAttributes instructor1ofCourse1 = typicalBundle.instructors.get("instructor1OfCourse1");
+        FeedbackSessionAttributes session = typicalBundle.feedbackSessions.get("session1InCourse1");
+        FeedbackQuestionAttributes typicalQuestion =
+                logic.getFeedbackQuestion(session.getFeedbackSessionName(), session.getCourseId(), 1);
+
+        loginAsInstructor(instructor1ofCourse1.getGoogleId());
+
+        String[] param = new String[] {
+                Const.ParamsNames.FEEDBACK_QUESTION_ID, typicalQuestion.getFeedbackQuestionId(),
+        };
+
+        FeedbackQuestionSaveRequest saveRequest = getTypicalTextQuestionSaveRequest();
+        FeedbackTextQuestionDetails textQuestionDetails = new FeedbackTextQuestionDetails();
+        textQuestionDetails.setRecommendedLength(-1);
+        saveRequest.setQuestionDetails(textQuestionDetails);
+        SaveFeedbackQuestionAction a = getAction(saveRequest, param);
+
+        assertThrows(InvalidHttpRequestBodyException.class, () -> {
+            getJsonResult(a);
+        });
+
+        // question is not updated
+        assertEquals(typicalQuestion.getQuestionDescription(),
+                logic.getFeedbackQuestion(typicalQuestion.getId()).getQuestionDescription());
```

Comment:
For this test case, I had set the recommended length to be -1 so that it throws an exception.
`textQuestionDetails.setRecommendedLength(-1);` 
should I add an assertEquals check for that?

Reply:
> should I add an assertEquals check for that?

Asserts that what equals to what?
If the question recommended length is updated to `-1` after the process, is it not a sign that your validation does not perform its job?

Reply:
the question recommended length is not changed during the running of the test case. I had set it to -1 befor `SaveFeedbackQuestionAction a = getAction(saveRequest, param);` so that the exception is thrown. However, the recommended length does not change after the process

Reply:
@RonakLakhotia, you added a check for the `textQuestionDetails` object which you created yourself. This check is meaningless. What you should check against is the question object that you get from the back end.

Reply:
sorry for the misunderstanding, got a bit confused